### PR TITLE
feat: Add @Kaanan2000 as a contributor

### LINKED ISSUE #293: Feature: Add contributor
### Description

As part of the Intro to Open Source course, we are encouraged to contribute to this repo.

### Suggested solution

Add myself as a contributor.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Getting Started Instructions

- [X] I agree to follow this project's Getting Started instructions

### PR DESCRIPTION
**Pull request by Kaanan2000**
## Description
This PR adds  @Kaanan2000 as a contributor.

## What type of PR is this? (check all applicable)

- [X] 🤝 Add a contributor
- [ ] 📝 Documentation Update

## Related Issues
Closes #293 

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.

- [X] ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- [X] ✅ Yes
- [ ] ❌ No

## Added to documentation?

- [X] 📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)
contributor_ss

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +36,9 @@ All contributors are required to abide by our [Code of Conduct](https://github.c
 ## All Contributors
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![All Contributors](https://img.shields.io/badge/all_contributors-129-orange.svg?style=flat-square)](#contributors-)
+
```
Comment (adiati98) [inline README.md:41]: You haven't run the `npm run contributors:generate`.

Also please remove the new lines (line 39 & 41).

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @Kaanan2000,

I see here that you've tried to resolve the merge conflicts.
Good job!!! 🎉

However, I have some reviews here for you to fix.

Please let me know once you've fix them by tagging me in the comment and I'll review your PR. 👍🏽 

Thank you.

**Comment by Kaanan2000** [comment]
Hi @adiati98 ,
I have removed the new lines and run the `npm run contributors:generate` command.
Plz let me know if I'm missing anything.

**Review by adiati98 — APPROVED** [review]
Well done, @Kaanan2000!
Thank you for the fixes! 😄

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕 

git merge GIF

**Pull request merged**